main-app/tests/directives/directive-tests.js

```diff
@@ -0,0 +1,30 @@
+
+(function(){
+    'use strict';
+    describe('Directive tests',function() {
+        var compile;
+        var rootScope;
+
+        beforeEach(
+            module(function($provide) {
```

Comment:
This beforeEach isn't doing anything useful AFAICT.
It's telling the dependency injection framework to provide the inner function when "tombola.noughtsAndCrosses.directive" is requested, but "tombola.noughtsAndCrosses.directive" looks like a module name rather than a provider. Do you need to provide a filter?